config.dist.json

```diff
@@ -1,6 +1,5 @@
 {
-  "nacelleSpaceId": "",
   "nacelleSpaceToken": "",
-  "nacelleSpaceIds": [],
-  "nacelleSpaceTokens": []
+  "nacelleSpaces": [],
```

Comment:
Replaced spaceIds and spaceTokens with Spaces to match what was in the documentation